WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -136,6 +135,8 @@ - (void)viewWillAppear:(BOOL)animated
     self.resultsController.delegate = self;
     [self.resultsController performFetch:nil];
     [self.tableView reloadData];
+    [self updateEditButton];
+    [self maybeShowNUX];
```

Comment:
I'm not specially happy about this for two reasons:
- I don't think the blog list should be the one telling the app delegate when it has to show NUX.
- This is duplicated on the results controller delegate because `controllerDidChangeContent:` is not called on `performFetch:`

I'm not sure what's the best way to do this but it doesn't feel well designed